implementation/parallel-to-sns-scatter-gather/scatter_gather/lambda_/responder/app.py

```diff
@@ -27,43 +27,46 @@
 logging.basicConfig(format='%(asctime)s %(levelname)-8s %(message)s', level=logging.INFO, datefmt='%Y-%m-%d %H:%M:%S')
 logging.getLogger().setLevel(logging.INFO)
 
-
+# get environment variables
 BASE_RATE = os.getenv('base_rate')
 VENDOR = os.getenv('vendor')
 
-
+# function generates the price quote for the rental company(vendor)
 def generate_quote(body):
     message_body = body
     daily_charge = int(BASE_RATE)
     logging.info(f"message_body (before): {message_body}")
-    logging.info(f"message_body[data]: {message_body['data']}")
+    # create price quote based on days of rental
     days = int(message_body['data']['days_rental'])
     message_body['data']['price_quote'] = daily_charge * days
     message_body['data']['vendor'] = VENDOR
     logging.info(f"message_body (after): {message_body}")
     return message_body
 
+# lambda function receives quote request from the customer and generates the price quote.
+# lambda functions supports both solutions step function and sns.
 def lambda_handler(event, context):
-    # Print the event object to the logs
-    logging.info("Received event: " + json.dumps(event, indent=2))
-    # if SQS_QUEUE_URL is not None:
+
+    logging.debug("Received event: " + json.dumps(event, indent=2))
+    # sns use case
     if 'Records' in event:
         for record in event['Records']:
             if 'Sns' in record:
```

Comment:
Could this be refactored to extract a get_body method? It'll make this method much simpler